pyQuARC/code/gcmd_validator.py

```diff
@@ -83,15 +83,16 @@ def _download_files(force=False):
         if gcmd_date.date() < current_datetime.date() or force:
             try:
                 for keyword, link in GCMD_LINKS.items():
-                    print(f"Downloading {keyword} file from GCMD ...")
-                    response = urllib.request.urlopen(GCMD_LINKS[keyword])
+                    # Downloading updated gcmd keyword files
+                    response = urllib.request.urlopen(link)
```

Comment:
Why not normal requests? why use urllib?

Reply:
i don't remember lol, i've updated it to use requests